Python/pdf_reorder/README.md

```diff
@@ -0,0 +1,32 @@
+
+## A pdf reorder or Rearranging Script
+
+#### I used the pdfrw library 
+It is built purely in python
```

Comment:
Kind of redundant. In the Python directory, everything is in Python

Reply:
Okay I will remove that too.